src/harness/testcases/WINNF_FT_S_IPR_testcase.py

```diff
@@ -250,12 +250,12 @@ def generate_IPR_2_default_config(self, filename):
     # Load Devices
     device_a = json.load(
         open(os.path.join('testcases', 'testdata', 'device_a.json')))
-    device_a['installationParam']['latitude'] = 30.71570
-    device_a['installationParam']['longitude'] = -88.09350
+    device_a['installationParam']['latitude'] = 30.23534
+    device_a['installationParam']['longitude'] = -87.93972
     device_b = json.load(
         open(os.path.join('testcases', 'testdata', 'device_b.json')))
-    device_b['installationParam']['latitude'] = 30.71571
-    device_b['installationParam']['longitude'] = -88.09351
+    device_b['installationParam']['latitude'] = 44.47642
```

Comment:
Please move device_b close to device_a so that they both impact the East3 DPA.

We should also have similar pairs of CBSDs for the other two DPAs (East21, West4); see the text of step 5. (I'm sorry I didn't realize that this also needed to be done when I was commenting on the issue!)

Reply:
ok, will fix this.